srcpkgs/strawberry/template

```diff
@@ -1,6 +1,6 @@
 # Template file for 'strawberry'
 pkgname=strawberry
-version=1.1.3
+version=1.2.3
 revision=1
 build_style=cmake
 configure_args="-DBUILD_WITH_QT6=ON -DQT_HOST_PATH=/usr"
```

Comment:
https://github.com/void-linux/void-packages/actions/runs/12260844918/job/34843784266?pr=53450#step:7:249

Reply:
Removed the redundant args

I'm sorry I didn't go through the build logs. Won't happen again